server/sonar-web/src/main/js/app/components/search/Search.js

```diff
@@ -297,10 +306,18 @@ export default class Search extends React.PureComponent {
     this.setState({ selected });
   };
 
+  handleClick = (event /*: Event */) => {
+    event.stopPropagation();
+  };
+
   innerRef = (component /*: string */, node /*: HTMLElement */) => {
     this.nodes[component] = node;
   };
 
+  searchInputRef = (node /*: HTMLInputElement | null */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)